src/main/MQ2ImGuiConsole.cpp

```diff
@@ -602,13 +612,14 @@ struct ImGuiZepConsole : public mq::imgui::ConsoleWidget, public mq::imgui::ImGu
 	std::shared_ptr<ZepConsoleSyntax> m_syntax;
 	int m_maxBufferLines = 10000;
 	bool m_autoScroll = true;
+	int m_fontSize = s_defaultFontSize;
 	std::string m_id;
 
 	ImGuiZepConsole(std::string_view id)
 		: m_id(std::string(id))
 	{
 		SetFont(Zep::ZepTextType::UI, mq::imgui::DefaultFont, 16);
-		SetFont(Zep::ZepTextType::Text, mq::imgui::ConsoleFont, 13);
+		SetFont(Zep::ZepTextType::Text, mq::imgui::ConsoleFont, m_fontSize);
 		SetFont(Zep::ZepTextType::Heading1, mq::imgui::DefaultFont, 28);
```

Comment:
I just noticed these other font setters.... Are these actually used? If they are, they should probably be configurable as well

Reply:
Yeah, I just watched the gif again. It looks strange to increase the log size but not the title bar and menu items. Some thought needs to go into the design of this, maybe these are derived from the newly selected font size.

You should probably write a function in console that does all of that for you given a selected font size and stop calling the zep SetFont directly (you can also wrap up the "is it changed" check there given how often we asked you to check first).

As far as the design, I think it's better to derive them, but maybe it's more desirable to set them all independently? Any thoughts on that @brainiac ?

Reply:
The other font setters aren't used by anything we've engaged the editor with so far, but it needs them or bad things could happen.

> It looks strange to increase the log size but not the title bar and menu items

It seems totally normal and fine to me. Those aren't part of the console impl so that wouldn't be the same set of changes anyhow. I don't recommend making that change, we're already lying about the font size to the editor -- what we SHOULD be doing is regenerating the font atlas, but we're not doing that here so it looks like crap when the font gets scaled up or down. It'll look worse for the smaller fonts of the UI

I think the simplest solution wins. this is simple. I don't want anything more complicated.

Reply:
Yeah i wanted to avoid messing with the font atlas and I figured when drawing your console for lua scripts you can always adjust the imgui font size that is not in the console already with WindowFontScale. 

I have updated the lua console example and will push that into this as well.

Reply:
No wonder it looked so bad, I thought these were a set of generated font atlas sizes. Why are the values hard coded then?

Reply:
> No wonder it looked so bad, I thought these were a set of generated font atlas sizes. Why are the values hard coded then?

no clue on that I just left them as is and changed the one I wanted to use

Reply:
These aren't actually changing the font. its stretching the font from its specified size.